rocky/OOI_database_seed.json

```diff
@@ -300,8 +300,9 @@
     "fields": {
       "last_updated": "2021-08-09T14:50:09.835Z",
       "data": {
-        "description": "The HTTP header X-DNS-Prefetch-Control is missing.",
+        "description": "The HTTP header X-DNS-Prefetch-Control is missing. The X-DNS-Prefetch-Control HTTP response header controls DNS prefetching, a feature by which browsers proactively perform domain name resolution on both links that the user may choose to follow as well as URLs for items referenced by the document.",
```

Comment:
This is a non-standard performance improving header, not a security issue. We should tell people this.
It could also be set by adding http-equiv tags in the html, which we currently do not account for.